src/roi/__tests__/ratio.test.ts

```diff
@@ -0,0 +1,21 @@
+test('ratio of ROI', () => {
+  const mask = testUtils.createMask([
+    [0, 1, 1, 1],
+    [0, 1, 0, 1],
+    [0, 1, 1, 1],
+    [0, 0, 0, 0],
+  ]);
+
+  expect(mask.ratio).toBe(4 / 4);
+});
+
+test('ratio of ROI', () => {
```

Comment:
Different tests should have different names.
The test name should be more precise about what it tests.